pwnagotchi/agent.py

```diff
@@ -258,7 +259,8 @@ def _find_ap_sta_in(self, station_mac, ap_mac, session):
         return None
 
     def _update_uptime(self, s):
-        secs = time.time() - self._started_at
+        p = subprocess.run(('cat /proc/uptime | awk \'{print $1}\''), shell=True, stdout=subprocess.PIPE)
```

Comment:
Why not read and parse /proc/uptime instead of running a process?

Reply:
yea I don't really know python... does it have an equivalent to awk?
feel free to make a pull request patching it